static/js/src/advantage/subscribe/react/components/Form/Version/OlderVersionModal.tsx

```diff
@@ -8,11 +8,15 @@ const OlderVersionModal = () => {
   return (
     <>
       <a onClick={openPortal} aria-controls="other-versions-modal">
```

Comment:
This link does not contain a `href` which makes it invalid and not a11y via keyboard.

Reply:
Nice catch! it's fixed